tests/installation/partitioning_firstdisk.pm

```diff
@@ -22,7 +22,18 @@ sub take_first_disk_storage_ng {
     return unless is_storage_ng;
     send_key $cmd{guidedsetup};    # select guided setup
     assert_screen 'select-hard-disks';
-    assert_and_click 'hard-disk-dev-sdb-selected';    # Unselect second drive
+    if (check_var('VIDEOMODE', 'text') || check_var('VIDEOMODE', 'ssh-x')) {
```

Comment:
Why not use assert and click on ssh-x? you have a working x session in ssh-x

Reply:
Yep, my mistake.  It should be for VIDEOMODE=text only. For yast installation with textmode, assert_and_click does not work.